client/test/TestingREADME.md

```diff
@@ -0,0 +1,85 @@
+#Intro
+
+This is Sam's testing README. 
+
+BE AWARE! Testing the front end is significantly more difficult than testing the backend. 
+
+You will need to replicate your actual production environment, and it can be very tricky. 
+
+TESTING IS FRAGILE. I have listed every change that I made below. Stuff will break, if you add/change stuff in our testing configuration, please document it below, as it may need modified in the future. 
+
+We are going to use Jest and Enzyme. 
+Jest is the actual Facebook testing suite for React, and allows us to take snapshot tests to ensure visual consistency when we are building our application. 
+
+Enzyme is used to test React Components. It is good for doing things such as ensuring that Component lifecyle methods are rendering correctly, ensuring that components are loading in the proper fashion, and ensuring that non-visual events such as click or on change events are properly firing. 
+
+Both of these packages will be set up to run via npm test. 
+
+sum.js && sum.test.js are just boilerplate tests to ensure that Jest is running. 

```

Comment:
Why do you need things that test if Jest is running? Are these two files really something you need?